examples/experimental/dagster-airlift/dagster_airlift/core/sensor/event_translation.py

```diff
@@ -88,8 +89,14 @@ def materializations_for_task_instance(
     dag_run: DagRun,
     task_instance: TaskInstance,
 ) -> Sequence[AssetMaterialization]:
-    mats = []
     asset_keys = airflow_data.asset_keys_in_task(dag_run.dag_id, task_instance.task_id)
+    return materializations_for_asset_keys_in_task_instance(dag_run, task_instance, asset_keys)
+
+
+def materializations_for_asset_keys_in_task_instance(
```

Comment:
nit: something about the "in_task_instance" language feels weird to me. Maybe corresponding, or mapped?